src/etc/htmldocck.py

```diff
@@ -429,7 +429,8 @@ def check_snapshot(snapshot_name, actual_tree, normalize_to_text):
     else:
         actual_str = flatten(actual_tree)
 
-    expected_str = expected_str.replace("{{channel}}", channel)
+    if expected_str is not None:
```

Comment:
Strangely enough, this one got unnoticed for quite some time. Since I needed to use `--bless` in here, the error appeared hence why it has this fix.

Reply:
It was fixed in https://github.com/rust-lang/rust/pull/98105 so I removed this commit. :)